### ISSUE TITLE
Main mepo hello should be more verbose

### ISSUE THREAD

**Issue by mathomp4** [GEOS-ESM/mepo#35]
The current mepo help message is a bit terse. This could be expanded (and maybe a man-page added?)

**Comment by yvikhlya** [comment]
@mathomp4 I agree that mepo help should be more verbose and explain what `mepo staus` abbreviations and color coding means.

**Comment by mathomp4** [comment]
> @mathomp4 I agree that mepo help should be more verbose and explain what `mepo staus` abbreviations and color coding means.

@yvikhlya Well it is an open-source project! So if you'd like to contribute... 😄 

But I'm going to try to get to this. My main annoyance is that argparse doesn't seem to allow for colored text, which would be nice. Still, I might be able to use something like `mepo help` which would just print a lot of text.
